README.md

```diff
@@ -27,12 +27,57 @@ The standardized MAFs generated by `standardize_mutation_data.py` will be writte
 
 **Sequence Source** ( `-s | --sequence-source` ) : The sequence source is a required field for a standard MAF as defined by the TCGA project. Accepted values are either WGS and WXS. For GENIE, WXS is used since GENIE samples are sequenced with targeted gene panels.
 
-**Annotation Suite Scripts Directory** ( ` -p | --annotation-scripts-home` ) : needs to point to the directory where the packaged Genome Nexus Annotation Pipeline JAR and annotation suite python scripts are located:
+**Annotation Suite Scripts Directory** ( ` -p | --annotation-scripts-home` ) : needs to point to the directory where the packaged Genome Nexus Annotation Pipeline JAR and annotation suite python scripts and data files are located:
 
-- merge_mafs.py
-- standardize_mutation_data.py
-- annotator.jar
+- merge_mafs.py (located in this repository)
+- standardize_mutation_data.py (located in this repository)
+- annotator.jar - The genome nexus annotation pipeline (See how to build the jar file [here](https://github.com/genome-nexus/genome-nexus-annotation-pipeline))
+- application.properties (optional) - custom properties for the annotation pipeline. The common use case is to specify custom genome nexus url with `genomenexus.base` property.
+- AwsSsl.truststore - custom trust store. Used to hold an ssl certificate (aka HTTPS certificate) of trusted websites. In our case the trust store holds certificates of custom genome nexus instance. See how to create trust store and upload the certificate [here](#create-a-trust-store).
 
+### Create a trust store
+
+Create an empty trust store with the following command:
+
+```
+keytool -noprompt \
+    -genkey -alias secure-key \
+    -dname "CN=annotation-tools" \
+    -storepass annotation-tools \
+    -keypass annotation-tools \
+    -keyalg RSA -keystore AwsSsl.truststore
+```
+
+You can export `.cer` file for your custom genome nexus using browser.
+There are multiple articles in the internet for different browsers and operating systems.
+
+After you have the `.cer` file, you can import it to the trust store with the command like the following (replace `CERTFILENAME` with the file name of your certificate):
+```
+keytool -import -noprompt \
+    -trustcacerts \
+    -alias CERTFILENAME \
+    -file CERTFILENAME.cer \
+    -keystore AwsSsl.truststore \
+    -storepass annotation-tools
+```
+
+### Install Python packages
```

Comment:
Maybe this should be moved up to the top of the README.txt ... It might be good to start this page with a more formal format like
Title (annotation-tools)
Description (This is a suite of tools which allow easy annotation of a number of genomic mutation source files to be annotated efficiently using Genome-Nexus. These tools leverage the genome-nexus-annotation-pipeline annotation tool, which is a compiled program for annotating a single source file containing mutations. The annotation-tools suite will employ that tool over any number of mutation sources files grouped by providing institutions.)
Requirements : ...